pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -145,12 +144,8 @@ func (p *criStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 		if !found {
 			ps = buildPodStats(podSandbox)
 			// Fill stats from cadvisor is available for full set of required pod stats
```

Comment:
For my education - what does this comment mean?

Reply:
CRI only provides part of cpu/memory stats summary api ,so we still need cadvisor stats to fill in the other fields in summary api.